erica/infrastructure/static/tax_offices.json

```diff
@@ -0,0 +1 @@
"}, {"name": "Finanzamt Dithmarschen", "bufa_nr": "2116"}, {"name": "Finanzamt Nordfriesland", "bufa_nr": "2117"}, {"name": "Finanzamt Itzehoe", "bufa_nr": "2118"}, {"name": "Finanzamt Kiel", "bufa_nr": "2120"}, {"name": "Finanzamt Lübeck", "bufa_nr": "2122"}, {"name": "Finanzamt Neumünster", "bufa_nr": "2124"}, {"name": "Finanzamt Ostholstein", "bufa_nr": "2125"}, {"name": "Finanzamt Plön", "bufa_nr": "2126"}, {"name": "Finanzamt Ratzeburg", "bufa_nr": "2127"}, {"name": "Finanzamt Rendsburg", "bufa_nr": "2128"}, {"name": "Finanzamt Eckernförde-Schleswig", "bufa_nr": "2129"}, {"name": "Finanzamt Stormarn", "bufa_nr": "2130"}, {"name": "Finanzamt Pinneberg", "bufa_nr": "2131"}, {"name": "Finanzamt Bad Segeberg Bewertung für Grundsteuer 71", "bufa_nr": "2171"}, {"name": "Finanzamt Eckernförde-Schleswig Bewertung fur Grundsteuer 72", "bufa_nr": "2172"}, {"name": "Finanzamt Elmshorn Bewertung für Grundsteuer 73", "bufa_nr": "2173"}, {"name": "Finanzamt Flensburg Bewertung für Grundsteuer 75", "bufa_nr": "2175"}, {"name": "Finanzamt Dithmarschen Bewertung für Grundsteuer 76", "bufa_nr": "2176"}, {"name": "Finanzamt Nordfriesland Bewertung für Grundsteuer 77", "bufa_nr": "2177"}, {"name": "Finanzamt Itzehoe Bewertung für Grundsteuer 78", "bufa_nr": "2178"}, {"name": "Finanzamt Kiel Bewertung für Grundsteuer 79", "bufa_nr": "2179"}, {"name": "Finanzamt Kiel Bewertung für Grundsteuer 80", "bufa_nr": "2180"}, {"name": "Finanzamt Nordfriesland Bewertung für Grundsteuer 81", "bufa_nr": "2181"}, {"name": "Finanzamt Lübeck Bewertung für Grundsteuer 82", "bufa_nr": "2182"}, {"name": "Finanzamt Dithmarschen Bewertung für Grundsteuer 83", "bufa_nr": "2183"}, {"name": "Finanzamt Neumünster Bewertung für Grundsteuer 84", "bufa_nr": "2184"}, {"name": "Finanzamt Ostholstein Bewertung für Grundsteuer 85", "bufa_nr": "2185"}, {"name": "Finanzamt Plön Bewertung für Grundsteuer 86", "bufa_nr": "2186"}, {"name": "Finanzamt Ratzeburg Bewertung für Grundsteuer 87", "bufa_nr": "2187"}, {"name": "Finanzamt Rendsburg Bewertung für Grundsteuer 88", "bufa_nr": "2188"}, {"name": "Finanzamt Eckernförde-Schleswig Bewertung für Grundsteuer 89", "bufa_nr": "2189"}, {"name": "Finanzamt Stormarn Bewertung für Grundsteuer 90", "bufa_nr": "2190"}, {"name": "Finanzamt Pinneberg Bewertung für Grundsteuer 91", "bufa_nr": "2191"}]}, {"state_abbreviation": "th", "name": "Thüringen", "tax_offices": [{"name": "Finanzamt Erfurt", "bufa_nr": "4151"}, {"name": "Finanzamt Ilmenau", "bufa_nr": "4154"}, {"name": "Finanzamt Eisenach", "bufa_nr": "4155"}, {"name": "Finanzamt Gotha", "bufa_nr": "4156"}, {"name": "Finanzamt Mühlhausen", "bufa_nr": "4157"}, {"name": "Finanzamt Sondershausen", "bufa_nr": "4159"}, {"name": "Finanzamt Gera", "bufa_nr": "4161"}, {"name": "Finanzamt Jena", "bufa_nr": "4162"}, {"name": "Finanzamt Pößneck", "bufa_nr": "4165"}, {"name": "Finanzamt Altenburg", "bufa_nr": "4166"}, {"name": "Finanzamt Sonneberg", "bufa_nr": "4170"}, {"name": "Finanzamt Suhl", "bufa_nr": "4171"}]}]}
```

Comment:
Out of curiosity: why is this part of the infrastructure?

Reply:
[Refactoring suggestion](https://steuerlotse.atlassian.net/browse/STL-2070?atlOrigin=eyJpIjoiM2U3MTAzZmZiOTEyNDMzYTk0ZmYyMzUyNGEyMmYxYzAiLCJwIjoiaiJ9)